src/routePaths.js

```diff
@@ -2,7 +2,8 @@ export default {
   establishmentRoll: '/establishment-roll',
   inToday: '/establishment-roll/in-today',
   iepHistory: '/offenders/:offenderNo/iep-details',
```

Comment:
Keeping old URL very briefly because new-nomis-ui needs changing to point to the new one as well.